loop/filestorage/aws.js

```diff
@@ -14,7 +14,7 @@ var CONTENT_TYPE = "application/json";
 function AwsDriver(options, settings, statsdClient) {
   this.statsdClient = statsdClient;
   this._settings = settings || {};
-  this._publicBucket = options.publicBucket || DEFAULT_PUBLIC_BUCKET;
+  this._publicBucket = settings.bucketName || DEFAULT_PUBLIC_BUCKET;
```

Comment:
It should be options and not settings here. because you settings are empty below.